solr/core/src/test/org/apache/solr/handler/export/TestExportWriter.java

```diff
@@ -28,14 +28,14 @@
 import java.util.Map;
 import java.util.Set;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.lucene.util.TestUtil;
 import org.apache.solr.SolrTestCaseJ4;
 import org.apache.solr.common.SolrInputDocument;
 import org.apache.solr.common.util.SuppressForbidden;
 import org.apache.solr.common.util.Utils;
 import org.apache.solr.request.SolrQueryRequest;
 import org.apache.solr.schema.SchemaField;
-import org.codehaus.jackson.map.ObjectMapper;
```

Comment:
Test used to use old Jackson class. Replaced with newer one and removed old Jackson pulled in by Hadoop.